custom/enikshay/tasks.py

```diff
@@ -523,7 +524,7 @@ def _get_fulfilled_vouchers(self):
         relevant_vouchers = [
             voucher for voucher in self._get_all_vouchers()
             if (voucher.get_case_property('voucher_type') == 'prescription'
-                and voucher.get_case_property('state') == 'fulfilled')
+                and voucher.get_case_property(DATE_FULFILLED))
```

Comment:
this looks good to me. I just checked, and this property only gets set when the state is set to fulfilled, so this is a good proxy for the voucher ever having been in the "fulfilled" state. 👍